org.eevolution.hr_and_payroll/src/main/java/base/org/eevolution/model/MHRProcess.java

```diff
@@ -1319,8 +1314,25 @@ private MHRMovement createMovement(MHRConcept concept, MHRAttribute attribute, b
 		movement.setIsManual(concept.isManual());
 		movement.setC_BP_Group_ID(businessPartner.getC_BP_Group_ID());
 		movement.setEmployee(employee);
+		if (!MHRConcept.TYPE_RuleEngine.equals(concept.getType())) {
+			int currencyPrecision = MCurrency.getStdPrecision(getCtx(), Env.getContextAsInt(p_ctx, "#C_Currency_ID"));
+			Optional<Integer> conceptStandardPrecisionOptional = Optional.ofNullable((Integer)concept.get_Value("StdPrecision"));
```

Comment:
@yamelsenih please change the concept.get_Value("StdPrecision") by the generate model